steps/lambdatest-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,72 @@
+title: LambdaTest Upload
+summary: |
+  Upload APK/IPA files to LambdaTest's App Automate
+description: |
+  Upload your build files to LambdaTest for remote device testing. Upload your IPA/APK file to LambdaTest App Automation and get back the app's url for your remote device tests
+website: https://github.com/LambdaTest/bitrise-step-lambdatest-upload
+source_code_url: https://github.com/LambdaTest/bitrise-step-lambdatest-upload
+support_url: https://github.com/LambdaTest/bitrise-step-lambdatest-upload/issues
+published_at: 2022-04-01T11:46:49.57948+05:30
+source:
+  git: https://github.com/LambdaTest/bitrise-step-lambdatest-upload.git
+  commit: 7cd2ff643722f2b89aa12fb2e3bd32a6ff60f56d
+project_type_tags:
+- ios
+- macos
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
```

Comment:
`curl` is missing from the dependency list. It is most probably preinstalled on Bitrise VMs and also probably all ubuntu and macOS installation but to be 100% safe it would be nice to list here.

Reply:
Done